src/datatable.js

```diff
@@ -219,6 +218,27 @@ export class DataTable {
   }
 
   displayValue(row, propertyName) {
-    return new Statham(row, Statham.MODE_NESTED).fetch(propertyName);
+    return fetchFrom(row, propertyName);
```

Comment:
It won't work with association columns. `propertyName` can be "car.color", and in that case `displayValue` should return `color`. Which won't be the case now I think.